src/exceptions/Bootloader-exception

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <exception licenseId="Bootloader-exception" name="Bootloader Distribution Exception">
+      <crossRefs>
+         <crossRef>https://github.com/pyinstaller/pyinstaller/blob/develop/COPYING.txt</crossRef>
```

Comment:
Do we have a policy on pinned vs. floating links for these references?  See discussion in github/choosealicense.com#555 for more on this, but if we only use a single link, I'd rather pin it to a specific version (like [this][1]) unless we are automatically scanning the floating target and confirming that it does still include the expected material (in this case, the exception text).

[1]: https://github.com/pyinstaller/pyinstaller/blob/v3.3/COPYING.txt#L16-L25

Reply:
no, we don't have a policy and it's usually (always?) been a regular link. I don't care either way and not sure we need an official policy. In this case, the text is at the top/easy to find. A pinned link could be useful if the file was long and the specific text hard to find. let's just leave this as is.

Reply:
> A pinned link could be useful if the file was long and the specific text hard to find.

Linking to a specific part of the file is convenient, and probably only a good idea for pinned links.  But I'm more concerned about “what happens after pyinstaller changes their license and drops the exception or moves the exception into a different file?”.  Projects don't relicense or rename their license files often, but it does happen.